### PR TITLE
Feature#234/implement frontend webhook

### PR DESCRIPTION
**Pull request by Alessandro100**
# Related Issue
- [234](https://github.com/DPigeon/Money-Tree/issues/234)

# Proposed changes
- added websocket support in the frontend

# Additional Info
- This story's scope is to only connect to the websocket

# Checklist (if applicable)
- [X] Tests

# Reviewer(s)
 - @DPigeon @EspressoCode

### CODE REVIEW

File: client/src/index.html [hunk 1]
```diff
@@ -6,6 +6,9 @@
     <base href="/" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
+    <script>
+      var global = global || window;
```
Comment (DPigeon) [inline client/src/index.html:10]: Just curious: why is this script needed?

**Review by DPigeon — APPROVED** [review]
Looks good! Just left 1 minor comment.

**Review by EspressoCode — APPROVED** [review]
Looks good, good job.

File: client/src/index.html (hunk 1 above)
Comment (Alessandro100) [reply client/src/index.html:10]: It's needed for one of the libraries we used. Added it more as a fix to an error

**Pull request merged**